EIPS/eip-5114.md

```diff
@@ -0,0 +1,66 @@
+---
+eip: 5114
+title: Soulbound Token
+description: A token that is attached to a "soul" at mint time and cannot be transferred after that.
+author: Micah Zoltu (@MicahZoltu)
+discussions-to: https://ethereum-magicians.org/t/eip-5114-soulbound-token/9417
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-05-30
+requires: 721
```

Comment:
Is this requirement necessary? I believe you can bind EIP5114 to any address?

Reply:
The EIP references EIP-721, so it requires it (not all editors agree on this policy).

Reply:
Hmm, in abstract it says that EIP5114 requires it to be bound to another NFT, and gives example of EIP721. In section "Backwards Compatibility" Its stated that EIP721 HAS to be used as a soul. I think the EIP should be more specific if EIP721 as a soul is mandatory, or is some other token or EIP also acceptable.

Reply:
Any EIP that matches the interface `ownerOf(uint256 id)` I think is the actual requirement.  EIP-721 is an example of that, as is EIP-5114.

Reply:
Does this mean that if EIP-5114 gets minted to an address that does not implement `ownerOf(uint256 id)`, then the EIP-5114 contract is no longer EIP-5114? Or can it be said that the contract is still EIP-5114 but the standard is not obeyed for this single instance?